st2common/st2common/openapi.yaml.j2

```diff
@@ -4927,7 +4927,7 @@ definitions:
       runner:
         $ref: '#/definitions/RunnerType'
       liveaction:
-        $ref: '#/definitions/LiveAction'
+        type: string
```

Comment:
double check this endpoint doesn't return the liveaction.  I think there was a unit test that I forced the liveaction to return embedded.  not sure if this is it.

Reply:
it was the aliasexecution endpoint I had to embedd the liveaction not this endpoint.  this endpoint returns the execution id now